tests/ethpm/validation/test_manifest.py

```diff
@@ -109,13 +91,11 @@ def test_extract_contract_types_from_deployments(data, expected):
     assert actual == expected
 
 
-@pytest.mark.parametrize("version", ("2"))
-def test_validate_manifest_version_validates_version_two_string(version):
-    validate = validate_manifest_version(version)
-    assert validate is None
+def test_validate_manifest_version_validates_version_three_string():
+    assert validate_manifest_version("ethpm/3") is None
 
 
-@pytest.mark.parametrize("version", (1, 2, "1" "3", b"3"))
+@pytest.mark.parametrize("version", (2, 3, "2" "3", b"3"))
```

Comment:
Is `ethpm/3` the only acceptable answer here?

Reply:
And should `"2"` and `"3"` be separated by a comma?

Reply:
> Is ethpm/3 the only acceptable answer here?
Yup, that's the only valid version string for an ethpm v3 manifest.